grails-app/controllers/au/org/ala/spatial/portal/LogController.groovy

```diff
@@ -0,0 +1,40 @@
+package au.org.ala.spatial.portal
+
+import grails.converters.JSON
+import org.apache.commons.httpclient.methods.StringRequestEntity
+import org.apache.http.client.methods.HttpPost
+import org.apache.http.client.methods.HttpGet
+
+class LogController {
+    def hubWebService
+    static allowedMethods = [index:'POST', search:'GET']
+
+    def index() {
+        String url = "${grailsApplication.config.layersService.url}/log/"
+        def json = request.JSON as Map
```

Comment:
Do we still need this line, if the only usage of `json` is in `new StringRequestEntity((json as JSON).toString()`? 

I mean if we can do `new StringRequestEntity((request.JSON as JSON).toString()` ?

Reply:
request.JSON .toString() works